TLM/TLM/UI/TrafficManagerTool.cs

```diff
@@ -1,24 +1,24 @@
 namespace TrafficManager.UI {
+    using System;
+    using System.Collections.Generic;
+    using System.Linq;
+    using System.Text;
+    using API.Manager;
+    using API.Traffic.Data;
```

Comment:
PLease return all imports to full qualified imports.
If you suggest an option to the auto-fixer, can you also commit that change?

Reply:
no these are just merge problems.